assets/js/components/ChecksResults.jsx

```diff
@@ -76,16 +72,14 @@ const ChecksResults = () => {
   const dispatchUpdateCatalog = () => {
     dispatch({
       type: 'UPDATE_CATALOG',
+      payload: { flat: '', provider: 'azure' }, //FIXME: Get provider properly
```

Comment:
This would translato toa query string like this 
```
/api/checks/catalog?flat=&provider=azure
```
I would make it `flat=true`, I find it a bit more robust.

what do you think?

Reply:
In the current scenario the queries like `flat&provider=azure` and `flat=&provider=azure`, both would work. I often time think that booleans are a tricky thing, because each language has their own syntax with `true|True|1` etc hehe.

I read that if the query param is a boolean having it without any parameter is pretty normal. But I don't have any issue changing it. What is the most normal thing out there?

Reply:
in elixir we can check the parameter directly since `nil` is a falsy value

Reply:
> In the current scenario the queries like `flat&provider=azure` and `flat=&provider=azure`, both would work
>
Yes, they would both work. I just find it a bit inconsistent and ambiguous as a contract.

> booleans are a tricky thing, because each language has their own syntax with true|True|1 etc
>
I see, it is also true that URLs should be independent from the language serving or consuming :smile: 
If then there is some language constraints in handling urls, that's another topic :smile: 

That said no big issue for me :+1:, anyway we do not yet have real API consumers to take care of.

Reply:
@fabriziosestito @nelsonkopliku At the end, what is the conclusion? Should I change it?

> in elixir we can check the parameter directly since nil is a falsy value
I'm not able to decipher your message @fabriziosestito XD